packages/runtime/src/components/builtin/Video/Video.tsx

```diff
@@ -2,8 +2,8 @@
 
 import { cx } from '@emotion/css'
 import { forwardRef, Ref, useEffect, useState } from 'react'
-import ReactPlayer from 'react-player'
 
+import { ReactPlayer } from '../../shared/react-player'
```

Comment:
Here and below, point imports to an intermediate file that provides the workaround.